docs/setting-up/server/docker.md

```diff
@@ -267,25 +267,28 @@ How to run PMM Server with Docker based on our [Docker image].
-| `DISABLE_UPDATES`          | Disables a periodic check for new PMM versions as well as ability to apply upgrades using the UI
-| `DISABLE_TELEMETRY`        | Disable built-in telemetry and disable STT if telemetry is disabled.
-| `METRICS_RESOLUTION`       | High metrics resolution in seconds.
-| `METRICS_RESOLUTION_HR`    | High metrics resolution (same as above).
-| `METRICS_RESOLUTION_MR`    | Medium metrics resolution in seconds.
-| `METRICS_RESOLUTION_LR`    | Low metrics resolution in seconds.
-| `DATA_RETENTION`           | How many days to keep time-series data in ClickHouse.
-| `ENABLE_VM_CACHE`          | Enable cache in VM.
-| `ENABLE_ALERTING`          | Enable integrated alerting.
-| `ENABLE_AZUREDISCOVER`     | Enable support for discovery of Azure databases.
-| `ENABLE_BACKUP_MANAGEMENT` | Enable integrated backup tools.
-| `PERCONA_TEST_SAAS_HOST`   | SaaS server hostname.
-| `PERCONA_TEST_DBAAS`       | Enable testing DBaaS features. (Will be deprecated in future versions.)
-| `ENABLE_DBAAS`             | Enable DBaaS features.
-| `PMM_DEBUG`                | Enables a more verbose log level.
-| `PMM_TRACE`                | Enables a more verbose log level including trace-back information.
-| `PMM_PUBLIC_ADDRESS`       | External IP address or the DNS name on which PMM server is running.
+| Variable                                                        | Description
+| --------------------------------------------------------------- | -----------------------------------------------------------------------
+| `DISABLE_UPDATES`                                               | Disables a periodic check for new PMM versions as well as ability to apply upgrades using the UI
+| `DISABLE_TELEMETRY`                                             | Disable built-in telemetry and disable STT if telemetry is disabled.
+| `METRICS_RESOLUTION`                                            | High metrics resolution in seconds.
+| `METRICS_RESOLUTION_HR`                                         | High metrics resolution (same as above).
+| `METRICS_RESOLUTION_MR`                                         | Medium metrics resolution in seconds.
+| `METRICS_RESOLUTION_LR`                                         | Low metrics resolution in seconds.
+| `DATA_RETENTION`                                                | How many days to keep time-series data in ClickHouse.
+| `ENABLE_VM_CACHE`                                               | Enable cache in VM.
+| `ENABLE_ALERTING`                                               | Enable integrated alerting.
+| `ENABLE_AZUREDISCOVER`                                          | Enable support for discovery of Azure databases.
+| `ENABLE_BACKUP_MANAGEMENT`                                      | Enable integrated backup tools.
+| `PERCONA_TEST_SAAS_HOST`                                        | SaaS server hostname.
```

Comment:
Let's even create a separate table for `PERCONA_TEST_` vars's and describe them separately ?

Reply:
https://jira.percona.com/browse/PMM-10078?focusedCommentId=297657&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-297657